tests/db/migration_tests/dump_tool.py

```diff
@@ -49,21 +49,21 @@ async def populate_facts_and_parameters(client, env_id: str):
             "id": "fact1",
             "source": const.ParameterSource.fact,
             "value": "value1",
-            "resource_id": "std::File[localhost,path=/tmp/test1]",
+            "resource_id": "std::testing::NullResource[localhost,name=test1]",
```

Comment:
maybe `internal` would be better for the agent name
```suggestion
            "resource_id": "std::testing::NullResource[internal,name=test1]",
```